packages/orchestration/src/exos/chain-hub.js

```diff
@@ -232,7 +232,9 @@ const ChainHubI = M.interface('ChainHub', {
     M.or(DenomDetailShape, M.undefined()),
   ),
   getDenom: M.call(BrandShape).returns(M.or(M.string(), M.undefined())),
-  makeChainAddress: M.call(M.string()).returns(CosmosChainAddressShape),
+  coerceCosmosAddress: M.call(AccountIdArgShape).returns(
+    CosmosChainAddressShape,
+  ),
```

Comment:
coerce*To*CosmosAddress?  From the name, I thought it might be taking a `CosmosChainAddress`, and producing something else. (maybe a CAIP-10?)

Reply:
Yes it's coercing to a value. Our naming scheme is to say `coerceFoo` for something that coerces to Foo. For example, https://github.com/Agoric/agoric-sdk/blob/b3ee3ddcb25fcc5b05f0b31a84f04a52d4a2f0ef/packages/governance/src/question.js#L52